schemas/system/domains.schema

```diff
@@ -25,5 +25,6 @@
   SystemType: Enum('Global', 'Autonomous', 'Standalone'),
   ServerType: Enum('Root', 'Server', 'Backup', 'Reserve'),
   NodeType: Enum('Master', 'Worker', 'Task'),
-  NodeStatus: Enum('Init', 'Active', 'Pause', 'Restart', 'Hang')
+  NodeStatus: Enum('Init', 'Active', 'Pause', 'Restart', 'Hang'),
+  Source: { type: 'string' }
```

Comment:
Should we maybe add this to [`metaschema` domains](https://github.com/metarhia/metaschema/blob/master/schemas/metaschema/domains.schema)?

Reply:
@tshemsedinov

Reply:
Sure, @nechaido, @belochub